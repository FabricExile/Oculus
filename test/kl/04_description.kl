require Oculus;

operator entry() {
  ovrDevice device();
  report(device.getDescription().Manufacturer);
}