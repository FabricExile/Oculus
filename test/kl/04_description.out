[FABRIC:MT] Oculus: Oculus Rift initialized.
[FABRIC:MT] Oculus VR
[FABRIC:MT] Oculus: Oculus Rift finalized.
